File: khelmel-js/src/models/bingo.player.js
class BingoPlayer {
    constructor(username, session) {
        this.username = username;
        this.session = session;
        this.ready = false;
    }

    getUsername() { return this.username; }
    getSession() { return this.session; }
    isReady() { return this.ready; }
    setReady(ready) { this.ready = ready; }
}

module.exports = BingoPlayer;

File: src/main/java/io/wulfcodes/khelmel/model/bingo/BingoPlayer.java
package io.wulfcodes.khelmel.model.bingo;

import io.javalin.websocket.WsContext;

public class BingoPlayer {
    private final String username;
    private final WsContext session;
    private boolean ready;

    public BingoPlayer(String username, WsContext session) {
        this.username = username;
        this.session = session;
        this.ready = false;
    }

    public String getUsername() { return username; }
    public WsContext getSession() { return session; }
    public boolean isReady() { return ready; }
    public void setReady(boolean ready) { this.ready = ready; }
}
